tableau-databricks/connection-resolver.tdr

```diff
@@ -24,16 +24,15 @@ limitations under the License.
 	
         <connection-normalizer>
             <required-attributes>
-                <setImpersonateAttributes/>
                 <attribute-list>
                     <attr>class</attr>
                     <attr>server</attr>
-                    <attr>dbname</attr>
+                    <attr>v-http-path</attr>
```

Comment:
is this a 1:1 replacement for dbname?

Reply:
Dbname is a reserved name in the connector dialog v2, so we have to use a vendor defined field. It is a 1:1 replacement.

Reply:
Will this change break compatibility with previous Tableau releases? It doesn't matter since it's tied to the Tableau release anyway ...